### PR TITLE
first step for setting up 2 column

### PR DESCRIPTION
**Pull request by straleyb**

### CODE REVIEW

**Comment by straleyb** [comment]
Screen Shot 2020-12-14 at 2 38 36 PM

File: app/views/hyrax/base/_show_collection_header.html.erb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <div class="col-sm-6">
   <% @collection = Collection.find(@presenter.member_of_collection_ids.first) %>
   <% @member_work_1 = Hyrax::SolrService.query("member_of_collection_ids_ssim:#{@presenter.member_of_collection_ids.first}", fl: 'id', rows: 3).first['id'] %>
-  <% @member_work_2 = Hyrax::SolrService.query("member_of_collection_ids_ssim:#{@presenter.member_of_collection_ids.first}", fl: 'id', rows: 3).second['id'] %>
-  <% @member_work_3 = Hyrax::SolrService.query("member_of_collection_ids_ssim:#{@presenter.member_of_collection_ids.first}", fl: 'id', rows: 3).third['id'] %>
+  <% @member_work_2 = Hyrax::SolrService.query("member_of_collection_ids_ssim:#{@presenter.member_of_collection_ids.first}", fl: 'id', rows: 3).first['id'] %>
```
Comment (CGillen) [inline app/views/hyrax/base/_show_collection_header.html.erb:4]: Should theses go back to `first`, `second`, `third`?

File: app/views/hyrax/homepage/_home_content.html.erb [hunk 2]
```diff
@@ -1,23 +1,70 @@
...
-      The collections are comprised of unique digitized and born-digital materials including photographs, articles, sheet music, manuscripts, ephemera, and more.
-      Oregon Digital is collaboratively managed by the Digital Scholarship Services of the University of Oregon Libraries and by Oregon State University Libraries.
-    </p>
+  <div class="col-md-5">
+    <% @collections = Collection.all[0..4] %>
+    <h2>FEATURED COLLECTIONS</h2>
+    <% unless @collections.to_a.empty? %>
+      <% @collections.each do |collection| %>
+        <% @member_work_1 = Hyrax::SolrService.query("member_of_collection_ids_ssim:#{collection.id}", fl: 'id', rows: 3).first['id'] %>
+        <% @member_work_2 = Hyrax::SolrService.query("member_of_collection_ids_ssim:#{collection.id}", fl: 'id', rows: 3).first['id'] %>
```
Comment (CGillen) [inline app/views/hyrax/homepage/_home_content.html.erb:10]: Same here? `first`, `second`, `third`?

File: app/views/hyrax/homepage/_home_content.html.erb [hunk 3]
```diff
@@ -1,23 +1,70 @@
...
+                { class: 'img-thumbnail', alt: '', height: 100, width: 100 },
+                'aria-hidden': true,
+                tabindex: -1
+              ) %>
+          </div>
+        <% end %>
+        <div class="col-sm-6">
+          <p class='collection-title'><h3><%= collection.title.first %></h3></p>
+          <p>
+            <h4><%= Hyrax::SolrService.query("member_of_collection_ids_ssim:#{collection.id}", fl: 'id', rows: 10000).length %> items
```
Comment (CGillen) [inline app/views/hyrax/homepage/_home_content.html.erb:50]: The collection is queried multiple times. I know `Hyrax::SolrService` does some caching, but should we query once and reference the results multiple times?
```
@member_docs = Hyrax::SolrService.query("member_of_collection_ids_ssim:#{collection.id}", fl: 'id', rows: 10000)
@member_works = @member_docs[0..2].each do |doc|
  ActiveFedora::Base.find(doc['id']).members.to_a.delete_if { |work| !work.class.to_s.include?('FileSet') }.first
end
@member_count = @member_docs.length
```
Or something to that effect?

**Review by CGillen — CHANGES_REQUESTED** [review]

**Review by CGillen — APPROVED** [review]

**Pull request merged**